### DISCUSSION TITLE
[Question]: How to change image sent to frigate+ into false positive ?

### DISCUSSION THREAD

**Discussion by JohannCR** [blakeblackshear/frigate#11374] [Ask A Question] [answered]
### What is your question:

It's the second time I confirm an object when sending to frigate+ and notice this was a false postive afterward...
The first time I deleted the image in frigate+ but was not able to resend it after, like the popup says it's permanent ^^
Is there a way to resend it or to change it into a false positive ?

**Comment by NickM-27** [comment, marked as answer]
there is no way to adjust this after sending it incorrectly

**Reply by JohannCR** [reply]
Too bad... May could be a useful feature in the future ?
Thx for your answer

**Comment by JohannCR** [comment]
FYI for those interested, the best workaround is to default to a false positive (by clicking on "this is not a **object**") if there is the slightest doubt. Because if wrong, it is possible to remove the false positives box. Not possible the other way around.
